shub/image/__init__.py

```diff
@@ -0,0 +1,24 @@
+import click
+import importlib
+
+
+@click.group(help="Release project with Docker")
```

Comment:
Yeah, naming is the most complex problem as usual 😄 
Maybe `Manage project based on custom Docker image`?